network/netplay/netplay_discovery.c

```diff
@@ -90,6 +91,7 @@ bool init_netplay_discovery(void)
 {
    struct addrinfo *addr = NULL;
    int fd = socket_init((void **) &addr, 0, NULL, SOCKET_TYPE_DATAGRAM);
+   char s[NETPLAY_HOST_STR_LEN];
```

Comment:
Do we know if inet_ntop_compat() already terminates the string for us on every platform? Might want to zero this out just in case.

Reply:
ah actually this were just for logging, there is no need for this, gonna remove it.

Reply:
it's binding to 0.0.0.0 so it should be fine always